text/3417-dropck-eyepatch-v3.md

```diff
@@ -0,0 +1,234 @@
+- Feature Name: (`dropck_eyepatch_v3`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Cleanup the rules for implicit drops by splitting `#[may_dangle]` into two separate attributes:
+`#[only_dropped]` and `#[fully_ignored]`. Change `PhantomData` to get completely ignored
+by dropck as its current behavior is confusing and inconsistent.
```

Comment:
these names are bikeshedable :sparkles: 

don't care too much about the specific names here